externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpBase.td

```diff
@@ -0,0 +1,30 @@
+#ifndef TORCH_MLIR_DIALECT_TCP_BASE
+#define TORCH_MLIR_DIALECT_TCP_BASE
+
+include "mlir/IR/OpBase.td"
+
+def Tcp_Dialect : Dialect {
+  let name = "tcp";
+  let cppNamespace = "::mlir::torch::tcp";
+  let description = [{
+    Tensor Compute Primitives (TCP) dialect.
+
+    TCP is a mid-level dialect in MLIR that enables transformations that
+    complement those in existing dialects, by supporting ops at a
```

Comment:
This isn't complete IMO -- we also supports ops at a lower level of abstraction than, say, the Torch dialect.  This also doesn't mention anything about the domain (ML & ML adjacent domains).

My 2c is to keep the description here bold and brief:

> TCP is a mid-level transformation oriented IR for deep learning & similar applications.

Reply:
Done.